### PR TITLE
Localization for copilot webview strings

### PR DESCRIPTION
**Pull request by amitjoshi438**
Added localization for copilot and notification panel.

### CODE REVIEW

**Review by BidishaMS — APPROVED** [review]

**Review by petrochuk — CHANGES_REQUESTED** [review]
XSS alert

**Comment by amitjoshi438** [comment]
Dismissed as false positive, they are  constant string not user inputs.

File: src/common/copilot/assets/scripts/copilot.js [hunk 1]
```diff
@@ -127,7 +127,7 @@
     copyButton.innerHTML = clipboardSvg;
     copyButton.classList.add("action-button");
     copyButton.classList.add("copy-button");
-    copyButton.title = "Copy to clipboard";
+    copyButton.title = copilotStrings. COPY_TO_CLIPBOARD_MESSAGE;
```
Comment (tehcrashxor) [inline src/common/copilot/assets/scripts/copilot.js:130]: breaking typo - space in `copilotStrings. COPY_TO_CLIPBOARD_MESSAGE;`

Comment (amitjoshi438) [reply src/common/copilot/assets/scripts/copilot.js]: Thanks! fixed

**Review by petrochuk — APPROVED** [review]

**Pull request merged**